src/sagemaker/image_uris.py

```diff
@@ -45,6 +45,8 @@ def retrieve(
     training_compiler_config=None,
     model_id=None,
     model_version=None,
+    tolerate_vulnerable_model=None,
+    tolerate_deprecated_model=None,
```

Comment:
could you default these two `bool` to `False`?

Reply:
I'd rather keep this `None`, keeping the pattern of this function.

Reply:
+1 to Janathan's comment, is there a specific reason to keep the pattern if this function intact?